src/sentry/conf/server.py

```diff
@@ -901,6 +901,9 @@ def create_partitioned_queues(name):
     "organizations:improved-search": False,
     # Enable incidents feature
     "organizations:incidents": False,
+    # Flags for enabling CdcEventsDatasetSnubaSearchBackend in getsentry. No effect in sentry
```

Comment:
nit, but I would prefer `sentry.io` in our open source repo

so it would read "Flags for enabling CdcEventsDatasetSnubaSearchBackend in sentry.io. No effect in open-source sentry at the moment.